CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 
+* [CHANGE] Remote-Write: Ability to set User-Agent #15201
```

Comment:
This is an ENHANCEMENT if it defaults to the same as before.
(and if not, please explain why not)

Reply:
Have modified to be an `[ENHANCEMENT]`.